pages/clusters/manage_queues.md

```diff
@@ -31,15 +40,20 @@ To create a new queue using the Buildkite interface:
 1. Select **Agents** in the global navigation to access the **Clusters** page.
 1. Select the cluster in which to create the new queue.
 1. On the **Queues** page, select **New Queue**.
-1. Enter a **key** for the queue, which can only contain letters, numbers, hyphens, and underscores, as valid characters.
+1. In the **Create a key** field, enter a unique _key_ for the queue, which can only contain letters, numbers, hyphens, and underscores, as valid characters.
```

Comment:
```suggestion
1. In the **Create a key** field, enter a unique _key_ for the queue, which can only contain letters, numbers, and underscores, as valid characters.
```

hyphens are not permitted.

Reply:
They are when you're creating a queue, so this should remain, and it reflects what's currently in the UI.